src/Verify.MSTest.Tests/Tests.cs

```diff
@@ -1,14 +1,29 @@
-﻿// ReSharper disable UnusedParameter.Local
+﻿// TODO: Add a test that uses the base class to prevent regressions until it's deleted
+// TODO: Add a test for when the [TestClass] already has a TestContext property
+
+#region WillBeSourceGenerated
+partial class Tests
```

Comment:
Reviewer note: In order to get feedback sooner, I manually generated what the source generator will.